activesupport/lib/active_support/core_ext/hash/slice.rb

```diff
@@ -18,8 +18,9 @@ def slice!(*keys)
 
   # Removes and returns the key/value pairs matching the given keys.
   #
-  #   { a: 1, b: 2, c: 3, d: 4 }.extract!(:a, :b) # => {:a=>1, :b=>2}
-  #   { a: 1, b: 2 }.extract!(:a, :x)             # => {:a=>1}
+  #   hash = { a: 1, b: 2, c: 3, d: 4 }
+  #   hash.extract!(:a, :b, :x) # => {:a=>1, :b=>2}
```

Comment:
I think we should drop `:x` here, to make it even clearer that `extract!` is the inverse of `slice!`.

The `:x` example was added in https://github.com/rails/rails/pull/7007 when the behaviour was changed, but the previous version was basically a bug, and 8 years on I don't think we need to continue documenting that `extract!` does what you would expect.

Reply:
@eugeneius Thanks for the feedback. I applied your suggestions.